README.md

```diff
@@ -1,36 +1,82 @@
-# n-gage
+# n-gage [![CircleCI](https://circleci.com/gh/Financial-Times/n-gage.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://github.com/Financial-Times/n-gage)
 
-<a href="https://docs.google.com/forms/d/e/1FAIpQLSf5InA7UJK9yNBCzidFKI_WNkfbl6of1eRlIACRspGXUcBx8A/viewform?usp=pp_url&entry.78759464=n-gage" target="_blank"><img src="https://i.imgur.com/UmScdZ4.png" alt="Yak button" border="0" align="right" width="150" title="Report a yak shaving incident for this repository"></a>
+`n-gage` is in every FT.com project and gives a standard set of `make` tasks and the `ngage` CLI to help set up, build and deploy projects.
 
-`n-gage` is in every Next project, giving a standard set of `make` tasks and `ngage` CLI to help with setting up, building and deployments.
 
-<br clear="right">
+## Requirements
 
-## Getting started
+* Node 8.x
 
-Starting a new repo?  You can do the following:
+
+## Installation
+
+```sh
+git clone git@github.com:Financial-Times/n-gage.git
+cd n-gage
+make install
+```
+
+## Development
+
+### Testing
+
+In order to run the tests locally you'll need to run:
 
 ```sh
-mkdir my-new-project
-cd my-new-project
-npm init -y
-npm install --save-dev --no-package-lock @financial-times/n-gage
+make test
 ```
 
-then create a new `Makefile` file with the following:
+### Install from NPM
+
+```sh
+npm install --save @financial-times/n-gage
+```
+
+### Usage
+
+Create a new `Makefile` file with the following:
 
 ```make
-# n-gage bootstrapping logic
+# This task tells make how to 'build' n-gage. It npm installs n-gage, and
```

Comment:
```suggestion
# This task tells `make` how to 'build' n-gage. It npm installs n-gage, and
```

You put 'make' in quotes for later mentions and I think it makes it easier to read.

**Update**: Just realised that this was copy-pasted from pre-existing text, but might be worthwhile making this change anyway?

Reply:
sounds good 👍